### PR TITLE
Luggage claim support!

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

File: src/main/java/com/hedvig/claims/graphql/Query.kt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.hedvig.claims.graphql
+
+import com.coxautodev.graphql.tools.GraphQLQueryResolver
+import org.springframework.stereotype.Component
+
+@Component
+class Query: GraphQLQueryResolver {
```
Comment (meletios) [inline src/main/java/com/hedvig/claims/graphql/Query.kt]: Do you need dat?

File: src/main/java/com/hedvig/claims/graphql/Mutation.kt [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    fun createLuggageClaim(
+            input: CreateLuggageClaimInput,
+            env: DataFetchingEnvironment
+    ): Boolean {
+
+        commandGateway.sendAndWait<Void>(
+                CreateLuggageClaimCommand(
+                        UUID.randomUUID().toString(),
+                        env.memberId,
+                        LocalDateTime.now(),
```
Comment (meletios) [inline src/main/java/com/hedvig/claims/graphql/Mutation.kt]: Should this come from the input object?

Comment (meletios) [reply src/main/java/com/hedvig/claims/graphql/Mutation.kt]: If not, is it the same with Instant.now?

File: src/main/java/com/hedvig/claims/aggregates/ClaimsAggregate.java [hunk 3]
```diff
@@ -362,4 +369,60 @@ public void on(EmployeeClaimStatusUpdatedEvent e) {
...
+
+    DataItem location = new DataItem();
+    location.date = e.getDate();
+    location.id = UUID.randomUUID().toString();
+    location.received = true;
+    location.type = ClaimDataType.DataType.TEXT;
+    location.userId = e.getMemberId();
+    location.title = "Place";
+    location.name = "Place";
+    location.value = e.getFrom() + " => " + e.getTo();
```
Comment (meletios) [inline src/main/java/com/hedvig/claims/aggregates/ClaimsAggregate.java]: Can we have two locations? Like one should be named destination and the other Origin?

File: src/main/java/com/hedvig/claims/graphql/Query.kt (hunk 1 above)
Comment (hedvigoscar) [reply src/main/java/com/hedvig/claims/graphql/Query.kt]: The GraphQL support gets really angry if it's not there 😅

File: src/main/java/com/hedvig/claims/graphql/Mutation.kt (hunk 2 above)
Comment (hedvigoscar) [reply src/main/java/com/hedvig/claims/graphql/Mutation.kt]: It is the same /kind of/. I think there is an argument for removing the timestamp - claims-service currently uses LocalDateTime for its concept of a date, I just figured timestamp could be fun to have in the event for later use.

Currently, we're only supporting the luggage claim being made through the UI at 'this date' - we could however add Date support later on in the input object. I think it's out of scope for now though 🙂

File: src/main/java/com/hedvig/claims/aggregates/ClaimsAggregate.java (hunk 3 above)
Comment (hedvigoscar) [reply src/main/java/com/hedvig/claims/aggregates/ClaimsAggregate.java]: Claims-service only understands the concept of a claim happening in one place today 😞 I went with this solution to fit how IEX works with this type of claim for now, but we can perhaps improve it in the future

**Comment by hedvigoscar** [comment]
I'm going to merge this for now so that I can test it in the staging environment.

**Pull request merged**